dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -413,6 +415,8 @@ def runValidation(self, jobId, interfaces):
             # Mark validation as finished in job tracker
             jobTracker.markJobStatus(jobId,"finished")
             errorInterface.writeAllRowErrors(jobId)
+            # Update error info for submission
+            jobTracker.getSubmissionErrorInfo(submissionId)
```

Comment:
Yes, given the possibility of updating a single file in a submission, this seemed the best way of keeping the number of errors and warnings accurate during updates.